src/Query/Expression.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace ClickHouseDB\Query;
+
+/**
+ * Query expression
+ *
+ * @package ClickHouseDB
+ */
+class Expression
+{
+    /**
+     * @var string
+     */
+    protected $expression = '';
+
+    /**
+     * Expression constructor.
```

Comment:
This comment can be removed

Reply:
phpdoc should be on every method IMHO, but it's your code, so it's up to you to decide.

Reply:
Actually, I can see phpdoc at __constructor for `ClickHouseDB\Query\Query`. Are you sure it should be removed? :)

Reply:
It's @smi2's code actually ;) I just try to contribute especially in direction of cleanup and refactorings.

But there's no reason to say it's constructor with PHPDocs when code says that for you

```php
class Expression {
    public function __constructor
```
already says the same as comment `* Expression constructor.`

You should add PHPDoc only when it adds value, not just visual spam. Code should be self explanatory if possible. Like Uncle Bob says: _Software code should read like well-written prose_

Reply:
> Actually, I can see phpdoc at __constructor for ClickHouseDB\Query\Query. Are you sure it should be removed? :)

Yes, I did not manage to clean it up everywhere yet :(

Reply:
I agree. But I also have a counter for that :) When everything have docblock - it adds rhythm, so code is easer to read, visually separate into blocks(visually extract methods, in this case).
And after I've added example into `__constructor` docblock not sure how to rewrite this block. Just remove `Expression constructor.` line would be good?

Reply:
That rythm is there as well when everything has no docblock ;). The visual noise is reduced. It also helps that we have recently integrated code style checks so it all then feels visually unified (it's not applied everywhere yet). Those visually separated blocks are methods themselves already. 

Soon there won't be need for comments at all as typed properties are coming with PHP 7.3 and typed arrays are planned too. So comments will make really sense only if they explain something. Currently they're usually used to specify what type is in arrays.

And yes, removing `Expression constructor` is good enough as the rest of the comment adds value. It helps the developer to understand how to use the class.

Also `@param string $expression` can be removed, that's already in method's signature